tracardi/process_engine/action/v1/password_generator_action.py

```diff
@@ -0,0 +1,142 @@
+from pydantic import validator
+
+from tracardi.service.plugin.runner import ActionRunner
+from tracardi.service.plugin.domain.result import Result
+from tracardi.service.plugin.domain.register import Plugin, Spec, MetaData, Documentation, PortDoc, Form, FormGroup, \
+    FormComponent, FormField
+from tracardi.service.plugin.domain.config import PluginConfig
+
+from password_generator import PasswordGenerator
+
+
+class Config(PluginConfig):
+    max_length: int
+    min_length: int
+    uppercase: int
+    lowercase: int
+    special_characters: int
+
+    @validator("min_length")
+    def check_min_max_value(cls, value, values):
+        if value > values["max_length"]:
+            raise ValueError("Minimum length is bigger than maximum length")
```

Comment:
f"Minimal length `{value}` cannot be bigger than given maximal length `{values['max_length']}`"